eZ/Publish/Core/FieldType/Tests/RichText/Converter/Xslt/EzxmlToDocbookTest.php

```diff
@@ -23,20 +24,20 @@ protected function setUp()
     {
         parent::setUp();
 
-        if (!class_exists(Expanding::class)) {
+        if (!class_exists(ExpandingToRichText::class)) {
```

Comment:
Should this test be moved to ezplatform-xmltext-fieldtype maybe? I don't think they are ever executed due to the dependency to ezplatform-xmltext-fieldtype....

Reply:
Yes please :)

Reply:
Once all my pending PR's has been merged, I'll make a PR where I move the tests